app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ApplicationRecord
+  validates_presence_of :username, :email
+
```

Comment:
If we want to add `gem 'email_validator` to the gemfile we can add the following in here to check emails: `validates :email, email: true, uniqueness: true`

Reply:
I added the email gem and a line to test for uniqueness of email in the user model